source/content/overview/matrix.md

```diff
@@ -0,0 +1,56 @@
+# Matrix construction
+
+We normally use Brightway to do life cycle assessment calculations using matrices. This section describes how matrices are constructed. In a standard calculation, we would have three matrices:
+
+* `technosphere` matrix: Our model of the supply chain. It has columns of processes, and rows of products. Chimaera process+product nodes will have both a row and a column.
+* `biosphere` matrix: The interactions our processes have with the natural environment - either consumption of resources, or emissions to the natural environment. It has columns of processes (in the same order and dimension as the `technosphere` matrix), and rows of biosphere flows.
+* `characterization` matrix: The unit conversion from physical amounts of biosphere flows to the unit of damage. It has columns and rows of biosphere flows (same order and dimension as the `biosphere` matrix), and only has values along the diagonal.
+
+## Processed datapackages
+
+To speed up calculations, Brightway maintains a separate cache of the numerical data. When a change is made to a `Database`, it is marked as `dirty` in its metadata. When a calculation is initiated, the cache of all dirty databases is refreshed from the relational data store.
+
+## Positive and negative values in the `technosphere`
+
+You may have seen the following matrix equation for IO or LCA: $h = (I - A)^{-1}f$. Brightway **does not** use this equation - instead, we use the following: $h = A^{-1}f$. In other words, we do not assume that each column in our technosphere matrix (**A**) is normalized to one unit of production, nor do we assume that rows are in the same order as columns. These are arbitrary restrictions, and Brightway does not like arbitrary restrictions. So we choose to manually construct the technosphere matrix, and choose which numbers are positive and which are negative. The sign convention is:
+
+* Positive numbers in the technosphere are products (goods and services) being *produced* by a process. This can include waste.
```

Comment:
Needs to be in a red, bold, blinking admonition box. I recommend [the `attention` type](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html#admonition-types).